inference-engine/tests/functional/plugin/shared/src/single_layer_tests/scale_shift.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <debug.h>
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "single_layer_tests/scaleshift.hpp"
+
+namespace LayerTestsDefinitions {
+    std::string ScaleShiftLayerTest::getTestCaseName(const testing::TestParamInfo<ScaleShiftParamsTuple> &obj) {
+        std::vector<std::vector<size_t>> inputShapes;
+        InferenceEngine::Precision netPrecision;
+        std::string targetName;
+        std::vector<float> scale, shift;
+        std::tie(inputShapes, netPrecision, targetName, scale, shift) = obj.param;
+        std::ostringstream results;
+
+        results << "IS=" << CommonTestUtils::vec2str(inputShapes) << "_";
+        results << "Scale=" << CommonTestUtils::vec2str(scale) << "_";
+        results << "Shift=" << CommonTestUtils::vec2str(shift) << "_";
+        results << "netPRC=" << netPrecision.name() << "_";
+        results << "targetDevice=" << targetName << "_";
+        return results.str();
+    }
+
+    void ScaleShiftLayerTest::SetUp() {
```

Comment:
From the IE perspective, it is actually a single layer graph, but in fact, we strive to keep single layer tests with only one ngraph op. (Let's name them "single operation" from opset specification perspective).

I request to move such test shared `subgraph_tests`